src/Middleware/HttpLogging/src/HttpLoggingServicesExtensions.cs

```diff
@@ -12,6 +12,14 @@ namespace Microsoft.Extensions.DependencyInjection;
 /// </summary>
 public static class HttpLoggingServicesExtensions
 {
+    /// <summary>
+    /// Adds HTTP Logging services.
+    /// </summary>
+    /// <param name="services">The <see cref="IServiceCollection"/> for adding services.</param>
+    /// <returns>The original service collection for chaining.</returns>
+    public static IServiceCollection AddHttpLogging(this IServiceCollection services)
+    => services.AddHttpLogging(o => o = new HttpLoggingOptions());
```

Comment:
This overload should register the services and the other one should call it.
```suggestion
    {
        ArgumentNullException.ThrowIfNull(services);
        services.TryAddSingleton(ObjectPool.ObjectPool.Create<HttpLoggingInterceptorContext>());
        services.TryAddSingleton(TimeProvider.System);
        return services;
    }
```